mautic/mautic
# Ui fix of role edit form

| Q  | A
| --- | ---
| Bug fix? | 1
| New feature? | 0
| Related user documentation PR URL | null
| Related developer documentation PR URL | null
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2055
| BC breaks? | 0
| Deprecations? | 0

#### Description:
Some UI changes are described in the linked PR.

+ the user edit form opens when clicking a user's name in the user manager.

#### Steps to test this PR:
1. Edit a role, the issues should be gone.
2. Make sure that the user's name is clickable in the user manager and goes to the right edit form.

#### Steps to reproduce the bug:
1. Edit a role and you should be able to see the issues.
2. Check that the user edit form does not open by clicking the user's name in the user manager.
